beaker-benchmark.gemspec

```diff
@@ -5,12 +5,12 @@ require 'beaker-benchmark/version'
 Gem::Specification.new do |s|
   s.name        = "beaker-benchmark"
   s.version     = Beaker::DSL::BeakerBenchmark::Version::STRING
-  s.authors     = ["Puppetlabs"]
-  s.email       = ["silver-team@puppetlabs.com"]
+  s.authors     = ["Puppet, Inc."]
```

Comment:
Just spot checking, I am unable to find other gems that use "Puppet, Inc." as the author.  The specification for this is all over the map.  My preference would be just a simple "Puppet" as per the [bolt](https://rubygems.org/gems/bolt) and [beaker](https://rubygems.org/gems/beaker) gems.

Reply:
I based in on RAS: https://github.com/puppetlabs/ref_arch_setup/blob/master/ref_arch_setup.gemspec

However, if other projects are just using "Puppet" that seems best.